test/bench/100_pods/S3/bench_10k.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+#ArmoniK "2.11.0"
+export CONTROL_PLANE_URL=http://ab52de83041174907b555aabff366290-801613365.eu-west-3.elb.amazonaws.com:5001/
```

Comment:
Why is this hardcoded?

Reply:
ping?

Reply:
i will get it from the generated ouput json file !